.github/CODE_OF_CONDUCT.md

```
@@ -1,26 +1,46 @@
 
 ## Scope
-This Code of Conduct applies within all community spaces and also applies when an individual is officially representing the project in public spaces.
+
+This Code of Conduct applies within all community spaces and also applies when
+an individual is officially representing the project in public spaces.
 
 ## Enforcement
-Instances of abusive, harassing, or otherwise unacceptable behavior may be reported to the project maintainer(s) at [your-email@example.com]. All complaints will be reviewed and investigated promptly and fairly.
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the project maintainer(s) at \[your-email@example.com\]. All
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)